pathspider/cmd/upload.py

```diff
@@ -0,0 +1,284 @@
+
+import pycurl
+import argparse
+import logging
+import json
+import bz2
+import dateutil.parser
+import os
+
+from io import BytesIO
+
+BASELINK = "https://v3.pto.mami-project.eu/raw/"
+TOKEN = "APIKEY"
+
+def register_args(subparsers):
+    parser = subparsers.add_parser(name='upload',
+                                   help="Uploads data to PTO")
+
+    parser.add_argument("filename", help="Data file in .ndjson")
+    parser.add_argument("--campaign", help="Campaign the data belongs to")
+    parser.add_argument("--token", help="Authentification token")
+    parser.add_argument("--entry", nargs='+', help="Additional meta entry", metavar="ENTRY:VALUE")
+    #TODO: add this function
```

Comment:
In general, prefer making a new GitHub issue over hiding TODO comments in the code.

Reply:
Forgot to delete it after adding the function